src/containers/FeedContainer.ts

```diff
@@ -28,7 +28,7 @@ export const mapStateToProps = (state: AppState, ownProps: { navigation: TypedNa
         : feedPosts
         ;
     return {
-        onBack: () => ownProps.navigation.goBack(null),
+        onBack: () => ownProps.navigation.popToTop(),
```

Comment:
replace could be another choice here. In both cases functionality is kind of fragile :-/

Reply:
I don't necessary agree. Replace would be fragile, because it depends on the history and in our case we use FeedContainer on two tabs, so the two cases should be handled separately.

This just takes you to the toplevel, which is exactly what I wanted here. If we want something else we would have to touch this anyhow.

Reply:
I was not precise enough in my previous comment. Replace should be used when **navigating to** the FeedContainer. So the onBack could stay `navigation.goBack()`
This one is fragile in a sense that if you open a Feed elsewhere, the back button will take you somewhere 'unexpected'.